src/chrome/content/rules/SheldonBrown.com.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="SheldonBrown.com">
+	<target host="sheldonbrown.com" />
+	<target host="www.sheldonbrown.com" />
```

Comment:
cdn.sheldonbrown.com
cdn-4.sheldonbrown.com
cdn-6.sheldonbrown.com

Reply:
Added, thank you!